config/main.py

```diff
@@ -12,7 +12,7 @@
 import threading
 import time
 
-from minigraph import parse_device_desc_xml
+from minigraph import earse_device_desc_xml
 from portconfig import get_child_ports, get_port_config_file_name
```

Comment:
It appears that `get_port_config_file_name` is no longer needed now, either.

Reply:
fixed